leetcode_python/Tree/create-binary-tree-from-descriptions.py

```
@@ -0,0 +1,77 @@
+#     def __init__(self, val=0, left=None, right=None):
+#         self.val = val
+#         self.left = left
+#         self.right = right
+class Solution(object):
+    def createBinaryTree(self, descriptions):
+        nodes = {}
+        children = set()
+
+        def get(v):
+            if v not in nodes:
+                nodes[v] = TreeNode(v)
```

Ruff (0.16.1)

[error] Undefined name `TreeNode`

(F821)